ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/ExpectedWithdrawals.java

```diff
@@ -209,6 +245,108 @@ private static List<Withdrawal> getExpectedWithdrawals(
     return expectedWithdrawals;
   }
 
+  public void processWithdrawals(
+      final MutableBeaconState genericState,
+      final ExecutionPayloadSummary payloadSummary,
+      final SchemaDefinitionsCapella schemaDefinitionsCapella,
+      final BeaconStateMutators beaconStateMutators,
+      final SpecConfigCapella specConfigCapella)
+      throws BlockProcessingException {
+    final SszList<Withdrawal> expectedWithdrawals =
+        schemaDefinitionsCapella
+            .getExecutionPayloadSchema()
+            .getWithdrawalsSchemaRequired()
+            .createFromElements(withdrawalList);
```

Comment:
NIT: it looks like we are creating this list twice (here and in `processWithdrawalsUnchecked`). Why not pass the list of expected withdrawals to `processWithdrawalsUnchecked` as a param?